pkgs/development/python-modules/mkdocs-git-revision-date-localized-plugin/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, gitpython
+, mkdocs
+, pytz
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-git-revision-date-localized-plugin";
+  version = "1.2.0";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "timvink";
+    repo = "mkdocs-git-revision-date-localized-plugin";
+    rev = "v${version}" ;
+    hash = "sha256-l7NUmsdd1FHkbdqtRcMQBfplZJgLCXrN+QuYaQLpz0k=";
+  };
+
+  propagatedBuildInputs = [
+    gitpython
+    mkdocs
+    pytz
+  ];
+
```

Comment:
Please add `pytestCheckHook` to `nativeCheckInputs` or `doCheck = false` with a comment.

Reply:
Good idea, I added tests.